modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PersistentPageMemory.java

```diff
@@ -1381,7 +1420,8 @@ private Segment(int idx, DirectMemoryRegion region) {
                     pool.pages()
             );
 
-            maxDirtyPages = pool.pages() * 3L / 4;
+            dirtyPagesSoftThreshold = pool.pages() * 3L / 4;
```

Comment:
Is there a justification for choosing these values?

Reply:
No, these are educated guesses right now

Reply:
I don’t like this answer, but I can’t really object if we correct it later.